public_dropin_gpu_environments/python311_genai/env_info.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": "64d2ba178dd3f0b1fa2162f0",
```

Comment:
change ID

Reply:
and change environment version id

Reply:
to what version ?

Reply:
any ?

Reply:
python3 -c "import bson; print(bson.ObjectId())"

Reply:
There are fields `id` and `environmentVersionId`.
Every environment has an ID. It always stays the same, doesn't change with the environment bump.
When we do changes, we need to bump environment version ID, to make it a different version.

Just generate any ObjectId, using bson.